MANIFEST.in

```diff
@@ -30,3 +30,6 @@ recursive-include invenio_rdm_records *.png *.svg
 recursive-include invenio_rdm_records *.po *.pot
 recursive-include invenio_rdm_records *.yaml
 recursive-include tests *.py *.csv *.json
+
+# added by check-manifest
+recursive-include invenio_rdm_records *.py
```

Comment:
:+1: weird this was still missing by now :sweat_smile:

Reply:
i think so - when i run ```./run-tests.sh``` locally it suggested this.